src/_gettsim/parameters/lohnst.yaml

```diff
@@ -0,0 +1,34 @@
+---
+lohnst_einkommensgrenzen:
+  name:
+    de: Einkommensgrenzen für Steuerklasse 5 und 6
+    en: Thresholds for Steuerklasse 5 and 6
+  description:
+    de: >-
+      Einkommensgrenzen für Steuerklasse 5 und 6, innerhalb derer
+      Mindest- und Maximalsteuern festgelegt sind.
+    en: >-
+      Thresholds for Steuerklasse 5 and 6, where
+      minimum and maximum tax rates apply.
+  reference: § 39b Absatz 2 Satz 7 EStG
+  note:
+    de: >-
+      Nur für 2022 bis jetzt implementiert
```

Comment:
Wouldn't this render GETTSIM essentially unusable for years prior to 2022? I think we should expand this.

Reply:
parameters before 2022 are added